src/util/samplebuffer.h

```diff
@@ -1,13 +1,17 @@
-#ifndef MIXXX_UTIL_SAMPLEBUFFER_H
-#define MIXXX_UTIL_SAMPLEBUFFER_H
+#pragma once
```

Comment:
#pragma once 
looks really neat. I am only afraid if using it hits us hard later, using other utility tools on the codebase because it is not standard.  mmm ...

Reply:
Any comment here?

Reply:
I usually agree with the [C++ Core Guidelines](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rs-guards), but in this special case I disagree with their recommendation. All major compilers support this extension and they would never be able to remove it. The #pragma avoids to write redundant and unchecked code and it could easily be reverted back to #include guards (maybe even with just a shell script, find, grep, and sed?) if we should ever need it. There is no consistent rule how to name the guards and we already have many variants in Mixxx. Let's get rid of them gradually.